api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/OpenEmpiPatientUpdater.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.module.registrationcore.api.mpi.openempi;
+
+import org.openmrs.Patient;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiAuthenticator;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiPatientUpdater;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+public class OpenEmpiPatientUpdater implements MpiPatientUpdater {
```

Comment:
I don't see the purpose of having an entire class to have a single 2-line method, rather than having this be part of a bigger Connector class